BinariesCheck.py

```diff
@@ -96,6 +94,16 @@ def __init__(self, pkg, path, fname, is_ar, is_shlib):
         self.mktemp = False
 
         is_debug = path.endswith('.debug')
+        # Currently this implementation works only on specific
+        # architectures due to reliance on arch specific assembly.
+        if pkg.arch in ['armv6hl', 'armv7hl', 'aarch64']:
```

Comment:
should not be a list '[' ... ']' but a tuple '(' .. ')'

Reply:
changed to startswith('armv')